python/ray/autoscaler/aws/node_provider.py

```diff
@@ -212,7 +215,11 @@ def create_node(self, node_config, tags, count):
             "SubnetId": subnet_id,
             "TagSpecifications": tag_specs
         })
-        self.ec2.create_instances(**conf)
+
+        logger.info(f"NodeProvider: Calling create_instances {count}...")
+        L = self.ec2.create_instances(**conf)
```

Comment:
Can you avoid using the f prefix? I don't think that works in python 2. Instead, use plain {} and call .format() with the args.

Reply:
@ls-daniel Do you want to fix this so we can get this merged?